javaCreateAccountCore/src/main/java/com/hamdan/forzenbook/java/createaccount/core/di/JavaCreateAccountModule.java

```diff
@@ -0,0 +1,56 @@
+package com.hamdan.forzenbook.java.createaccount.core.di;
+
+import static com.hamdan.forzenbook.java.core.GlobalConstants.LOGIN_BASE_URL;
+
+import com.hamdan.forzenbook.java.createaccount.core.data.network.CreateAccountService;
+import com.hamdan.forzenbook.java.createaccount.core.data.repository.CreateAccountRepository;
+import com.hamdan.forzenbook.java.createaccount.core.data.repository.CreateAccountRepositoryImpl;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountUseCase;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountUseCaseImpl;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountValidationUseCase;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountValidationUseCaseImpl;
+
+import javax.inject.Named;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.hilt.InstallIn;
+import dagger.hilt.android.components.ViewModelComponent;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+@InstallIn(ViewModelComponent.class)
+@Module
+public class JavaCreateAccountModule {
+
+    final private String MODULE_NAME = "java_create_Account";
```

Comment:
Consistent casing please

Reply:
Also this doesn't seem to be naming a module, but naming the specific retrofit instance in here

Reply:
updated to reflect that